woocommerce/woocommerce
# Run promotions WP Cron event twice daily

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Increases frequency of cron job added in https://github.com/woocommerce/woocommerce/pull/51650 from daily to twice daily.



Closes 21774-gh-Automattic/woocommerce.com.
Related: https://github.com/woocommerce/woocommerce/pull/51650.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch.
2. Install [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/) in your WooCommerce environment, so you can view WP-Cron events.
3. You can inspect the action at `wp-admin/tools.php?page=crontrol_admin_manage_page&s=woocommerce_marketplace_cron_fetch_promotions`.

<p>
<img width="500" alt="image" src="https://github.com/user-attachments/assets/320b9abc-9814-40c8-b903-94e1056da994">
</p>

4. View the cron job. It should be set to run in 12 hours.
5. Please test around this change.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Increases frequency of cron job from daily to twice daily.

</details>
